common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2585,6 +2587,7 @@ StScrollBar {
 
 .user-widget.vertical .user-widget-label {
   @include fontsize($font-size * 1.5);
+  color: if($variant != 'lighter', lighten($osd_fg_color, 25%), darken($osd_fg_color, 12%));
```

Comment:
IMHO, the `.user-widget.vertical` elements should retain the unadjusted `$osd_fg_color` foreground color. This only makes them look inconsistent to me.

Reply:
IMHO, I think the modified color makes more sense, for 2 reasons:
1. The user name is "highlighted" in the login screen
2. More importantly, this ensures the user name is white (like panel button text) in the unlock screen for Arc/Arc-Dark variants, ensuring greater visual consistency (in the case of Arc-Lighter, bug remains due to upstream limitations)

Reply:
1. I can't think of any reason why the user should be highlighted.
2. You shouldn't break the consistency of the login dialog, just to work around upstream limitations to make the unlock dialog slightly less inconsistent. The situation with the unlock-dialog is what it is, and shouldn't be addressed in this manner, especially when there's an ongoing attempt for an upstream fix.

Please just use plain `color: $osd_fg_color;` here for now.

Reply:
Not liking it ... but done

Reply:
I understand your points and dissatisfaction with the current look using plain `$osd_fg_color`. My point is that applying special styling to individual elements is not the right thing to do. It's more important to have a uniform design for now, and any adjustments to that should be done in uniform fashion as well. I certainly want to improve the look once the upstream situation is clear, but for now I think this is the best thing to do.

Reply:
Not too frustrated :), I do see your point